todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,62 @@
+import requests, os, json, pymongo, dotenv
+from pymongo import MongoClient
+from requests import api
+from todo_app.data.todoitem import ToDoItem
+from requests.api import post
+from bson import ObjectId
+import certifi
+
+
+ca = certifi.where()
+connectionn_string = os.environ.get("PRIMARY_CONNECTION_STRING")
+client = MongoClient(connectionn_string, tlsCAFile=ca)
+db = client.todo_app
+todo_items = db.todo_tasks 
```

Comment:
This code executing outside a function means it runs as soon as the file is imported, which causes issues for your integration test! The easiest solution here might be to catch this in a function like we had to with environment variables back in M3:
```suggestion
def get_collection():
  ca = certifi.where()
  connectionn_string = os.environ.get("PRIMARY_CONNECTION_STRING")
  client = MongoClient(connectionn_string, tlsCAFile=ca)
  db = client.todo_app
  todo_items = db.todo_tasks 
  return todo_items
```
And then have other parts of this code call `get_collection()` rather than trying to use `todo_items` directly

Reply:
Done this for next commit.